File: LeetCode-JavaScript/09-回文数.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
     if (x < 0) {
        return false;
    }
    var sum = 0;
    var n = x;
    while (n != 0) {
        sum = sum *10 + n%10;
        n /= 10;
        n = Math.floor(n)
    }
    if (sum == x) {
        return true;
    }
    return false 
};

File: LeetCode-Go/09-回文数.go
package main
func isPalindrome(x int) bool {
	if x < 0 {
		return false;
	}
	sum := 0;
	n := x;
	for n != 0 {
		sum = sum *10 + n%10;
		n /= 10;
	}
	if sum == x {
		return true;
	}
	return false
}



